ci-automation/Dockerfile

```diff
@@ -1,12 +1,6 @@
-FROM ubuntu:18.04
+FROM hrishikesh/libvirtd
```

Comment:
@gbenhaim @ifireball agree with you.
@ifireball Not sure I understand the comment - "assuming we don't want our CI container because we don't need systemd, why can't we used kubevirt's?"
BTW, I have tried to use kubevirt/libvirt image with no avail. It uses lots of host resources which we are trying to avoid.

@ifireball I can contribute the code I come up with to the suggested repo by coping it. Right?

And another option is just copy the code from what I came up with to here.

What is the best approach?